packages/math-input/src/components/keypad-container.tsx

```diff
@@ -154,11 +154,17 @@ class KeypadContainer extends React.Component {
         // NOTE(charlie): We render the transforms as pure inline styles to
         // avoid an Aphrodite bug in mobile Safari.
         //   See: https://github.com/Khan/aphrodite/issues/68.
-        const dynamicStyle = {
+        let dynamicStyle = {
             ...(active ? inlineStyles.active : inlineStyles.hidden),
-            ...(!active && !hasBeenActivated ? inlineStyles.invisible : {}),
```

Comment:
TS doesn't like the possibility of `false` being part of `dynamicStyle`. I'm sure there's a fix for this, but in the meantime I just made it more explicit.